src/modules/kvs/kvs.c

```diff
@@ -2707,14 +2707,50 @@ static void process_args (struct kvs_ctx *ctx, int ac, char **av)
     }
```

Comment:
Commit message for f80b1a513481c21f53d2b194e33393bebaaa3d9b:

Maybe should read "...when the primary _namespace_ was checkpointed."

and "When checkpointing the primary _namespace_..."